tests/lib/src/common_utils.cpp

```diff
@@ -55,4 +55,54 @@ void fillBlobs(InferenceEngine::InferRequest inferRequest,
       throw std::logic_error("Input precision is not supported for " + item.first);
     }
   }
+}
+
+/**
+ * @brief Fill infer_request tensors with random values or image information
+ */
+void fillTensors(ov::InferRequest& infer_request, const std::vector<ov::Output<ov::Node>>& inputs) {
+    std::vector<std::pair<size_t, size_t>> input_image_sizes;
+
+    for (auto &input: inputs) {
+        if (input.get_shape().size() == 4) {
+            input_image_sizes.emplace_back(input.get_shape()[2], input.get_shape()[3]);
```

Comment:
some models have NHWC layout (for ex faster_rcnn_inception_v2_coco)
in this case input_image_sizes will contain WC, but HW is needed

Reply:
Fixed